File: app/src/main/java/com/ilustris/motiv/fragments/PicsFragment.java
package com.ilustris.motiv.fragments;


import android.media.Image;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

import androidx.cardview.widget.CardView;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.RecyclerView;

import com.bumptech.glide.util.Util;
import com.google.android.material.appbar.AppBarLayout;
import com.ilustris.motiv.R;
import com.ilustris.motiv.db.Picsdb;
import com.ilustris.motiv.tools.Utils;

import de.hdodenhof.circleimageview.CircleImageView;

/**
 * A simple {@link Fragment} subclass.
 */
public class PicsFragment extends Fragment {


    private AppBarLayout appbar;
    private CardView addicon;
    private RecyclerView recyclerview;

    public PicsFragment() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        View v = inflater.inflate(R.layout.fragment_pics, container, false);
        initView(v);
        return v;
    }


    private void initView(View v) {
        appbar = v.findViewById(R.id.appbar);
        addicon = v.findViewById(R.id.addpicard);
        ImageView pic = v.findViewById(R.id.pic);
        recyclerview = v.findViewById(R.id.recyclerview);
        Picsdb picsdb = new Picsdb(getActivity());
        picsdb.Carregar(recyclerview);
        addicon.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Utils.pickimage(getActivity());
            }
        });
        pic.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Utils.pickimage(getActivity());
            }
        });
    }
}

File: app/src/main/java/com/ilustris/motiv/fragments/PicsFragment.kt
package com.ilustris.motiv.fragments


import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView

import androidx.cardview.widget.CardView
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.RecyclerView

import com.google.android.material.appbar.AppBarLayout
import com.ilustris.motiv.R
import com.ilustris.motiv.model.Picsdb
import com.ilustris.motiv.tools.Utils

/**
 * A simple [Fragment] subclass.
 */
class PicsFragment : Fragment() {


    private var appbar: AppBarLayout? = null
    private var addicon: CardView? = null
    private var recyclerview: RecyclerView? = null


    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        // Inflate the layout for this fragment
        val v = inflater.inflate(R.layout.fragment_pics, container, false)
        initView(v)
        return v
    }


    private fun initView(v: View) {
        appbar = v.findViewById(R.id.appbar)
        addicon = v.findViewById(R.id.addpicard)
        val pic = v.findViewById<ImageView>(R.id.pic)
        recyclerview = v.findViewById(R.id.recyclerview)
        val picsdb = Picsdb(activity!!)
        picsdb.Carregar(recyclerview!!)
        addicon!!.setOnClickListener { Utils.pickimage(activity!!) }
        pic.setOnClickListener { Utils.pickimage(activity!!) }
    }
}// Required empty public constructor
